brayns/common/camera/Camera.h

```diff
@@ -231,7 +231,7 @@ class Camera : public BaseObject
     float _focalLength{0.f};
     float _fieldOfView{45.f};
 
-    CameraStereoMode _stereoMode{CameraStereoMode::none};
+    StereoMode _stereoMode{StereoMode::none};
```

Comment:
Since you're making changes, you should add another parameter to the stereo parameters in perspective cameras, which is the zero-parallax distance. The eye-separation is not enough to setup a correct stereo pair.

Reply:
Sure but this will go in a different PR.

Reply:
I was suggesting it because it was a good opportunity to address it, even if out of scope.

Reply:
I know, thanks for that. It's only that zero-parallax distance will need modifications in the ispc code, and it's probably going to require a bit of work. No time now :(